Setup travis build to publish tags to NPM (#12)
cerner/cucumber-forge-report-generator: RELEASE.md

# How to Release

This project uses [semantic-release](https://github.com/semantic-release/semantic-release) to automatically release new versions when commits are merged to the master branch.  The semantic-release process is triggered as part of the [Travis build](https://travis-ci.com/cerner/cucumber-forge-report-generator).

To ensure that releases are triggered properly, the following standards should be applied to commit messages for the master branch:

| Commit message                                                                                                                                                                                   | Release type               |

> |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------|

Reviewer: could we ditch some of the lines since they aren't needed in markdown?

Author: Yeah. sorry about that. Just lifted this directly from the semantic-release README and did not really thing to clean it up.

Releasing the project requires these steps:

0. Upgrade dependency versions as necessary. (See: `npm audit`)

> 1. Run `npm version *major|minor|patch*` (this project uses [semantic versioning](http://semver.org/))

Reviewer: ```suggestion
0. Run `npm version *major|minor|patch*` (this project uses [semantic versioning](http://semver.org/))
```

> 2. Push new commit to master and push the newly created tag (this will trigger a Travis build)

Reviewer: ```suggestion
0. Push new commit to master and push the newly created tag (this will trigger a Travis build)
```

> 3. Wait for the Travis build to complete

Reviewer: ```suggestion
0. Wait for the Travis build to complete
```

> 4. [Create a new release on GitHub](https://help.github.com/articles/creating-releases/) for the tag and add changelog information for the changes included in the release

Reviewer: ```suggestion
0. Upgrade dependency versions as necessary. (See: `npm audit`)
0. Run `npm version *major|minor|patch*` (this project uses [semantic versioning](http://semver.org/))
0. Push new commit to master and push the newly created tag (this will trigger a Travis build)
0. Wait for the Travis build to complete
0. [Create a new release on GitHub](https://help.github.com/articles/creating-releases/) for the tag and add changelog information for the changes included in the release
```

> 4. [Create a new release on GitHub](https://help.github.com/articles/creating-releases/) for the tag and add changelog information for the changes included in the release

Reviewer: ```suggestion
0. [Create a new release on GitHub](https://help.github.com/articles/creating-releases/) for the tag and add changelog information for the changes included in the release
```
